tokens/usecase/pricing/worker/pricing_worker_test.go

```diff
@@ -231,7 +231,9 @@ func (s *PricingWorkerTestSuite) TestGetPrices_Chain_FindUnsupportedTokens() {
 	// SHARK - CW pool (unsupported in tests but should be supported on mainnet)
 	// FURY - listed but no pools
 	// FURY.legacy - listed but no pools
-	s.Require().Equal(25, zeroPriceCounter)
+	//
+	// Update on May 13, 2024: 14 unsupported tokens because some tokens have been fallen back to backup pricining source Coingecko
+	s.Require().Equal(14, zeroPriceCounter)
```

Comment:
Nice. In the future, we will need to think about making this more dynamic so that when we update internal test state, these assertions would not break.

However, this is a future goal that is outside of scope of this PR - just noting

Reply:
👍  Agree. I don't like the magic number there as well